app/api/services/users/patch_user.py

```diff
@@ -3,15 +3,15 @@
 from api.db.models.user_models import Role, User
 from api.route.api_context import ApiContext
 from api.route.route_utils import get_or_404
+from api.route.schemas.user_schemas import UserPatchParams
```

Comment:
i noticed that we import `UserPatchParams` from `services` in the other file, but from `route` in this file. should we import it from `services`? This situation makes me realize an additional value of [google's python style guide on imports](https://google.github.io/styleguide/pyguide.html#22-imports), which suggests in most cases to do

```
from api.services import models
# use models.UserPatchParams
```

rather than

```
from api.services.models UserPatchParams
```

i think it helps avoid accidentally re-exporting classes i.e. i don't think it was intentional for the `user_schemas` module to be re-exporting `UserPatchParams`